vsintegration/src/FSharp.Editor/FSharp.Editor.fsproj

```diff
@@ -28,12 +28,23 @@
     <GeneratePkgDefFile>true</GeneratePkgDefFile>
     <IncludePkgdefInVSIXContainer>true</IncludePkgdefInVSIXContainer>
   </PropertyGroup>
+  <Import Project="$(FSharpSourcesRoot)\FSharpSource.targets" />
```

Comment:
I have no idea what this is, but it was added.

Reply:
I don't think any of this is necessary.  After commenting this out, it builds, and the VSIX that I installed into my VS appears to work.